processing/src/main/java/io/druid/query/extraction/MapLookupExtractor.java

```diff
@@ -43,12 +43,16 @@
 {
   private final Map<String, String> map;
 
+  private final boolean isOneToOne;
+
   @JsonCreator
   public MapLookupExtractor(
```

Comment:
This probably would've been a good place to create a new constructor that delegates to this 2-argument one.  There are a lot of lines changed just to handle defaulting `isOneToOne` to `false` where a second constructor would've resolve that.  No big deal, just commenting.